### DISCUSSION TITLE
Collections - reorder

### DISCUSSION THREAD

**Discussion by slowizzm** [processing/p5.js-web-editor#2663] [General]
New feature request

It would be wonderful if we can reorder sketches in our collections. Use case: semester lectures in sync with collection order.

**Comment by catarak** [comment]
Thank you for the suggestion! I think this issue points to something else that would be cool, which is having a "profile" like page that you could highlight a few specific collections in a specific order. Doesn't need to be anything super fancy! Would love to think more about this.

**Comment by Ajaya1000** [comment]
@catarak this npm   [react-dnd](https://github.com/react-dnd/react-dnd) package provides a drag n drop reordering functionality. 
And I also like the concept of having a profile page. One can share his/her profile link with others highlighting some of his/her sketches and some collections. Or maybe in the future, we can also add a **star** or **like** button which will be useful for independent online tutors/bloggers.

**Comment by bmoren** [comment]
Hello amazing people!
I'm jumping on to also request this feature request, it would be especially helpful in the teaching context to be able to move things into order of topics covered.
